src/plugins/cal-events/cal-events.js

```diff
@@ -37,14 +37,6 @@ var componentName = "wb-calevt",
 		if ( elm ) {
 			$elm = $( elm );
 
-			// Only initialize the i18nText once
-			if ( !i18nText ) {
-				i18n = wb.i18n;
-				i18nText = {
-					calendar: i18n( "cal" )
```

Comment:
Check if "cal" is used elsewhere, if not we will need to remove it from our CSV.

Reply:
It was moved to calendar.js line 38. No need to remove it from the CSV.